applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_explicit_solver.py

```diff
@@ -0,0 +1,115 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Import base class file
+from KratosMultiphysics.ConvectionDiffusionApplication import (
+    convection_diffusion_base_solver,
+)
+
+
+def CreateSolver(model, custom_settings):
+    return ConvectionDiffusionExplicitSolver(model, custom_settings)
+
+
+class ConvectionDiffusionExplicitSolver(
+    convection_diffusion_base_solver.ConvectionDiffusionBaseSolver
+):
+    """
+    The explicit class for convection-diffusion solvers.
+    See convection_diffusion_base_solver.py for more information.
+    """
+
+    def __init__(self, model, custom_settings):
+        # Construct the base solver and validate the remaining settings in the base class
+        super(ConvectionDiffusionExplicitSolver, self).__init__(model, custom_settings)
+
+        # Overwrite the base solver minimum buffer size
+        self.min_buffer_size = 2
+
+        element_name = self.settings["element_replace_settings"][
+            "element_name"
+        ].GetString()
+        if self.settings["use_orthogonal_subscales"].GetBool() is True:
+            if element_name in (
+                "QSConvectionDiffusionExplicit",
+                "DConvectionDiffusionExplicit",
+            ):
+                self.main_model_part.ProcessInfo.SetValue(
+                    KratosMultiphysics.OSS_SWITCH, 1
+                )
+            else:
+                err_msg = self.__class__.__name__
```

Comment:
I think this is not required. It will appear when the error pop up.

Reply:
Ok, removed.